### PR TITLE
chore: Update ToS and privacy policy

### LINKED ISSUE #2262: [Task]: Update privacy policy
### Task description

The privacy policy needs to be updated to include Sentry and Cloudflare

### Requirements

- [x] Update privacy policy in [Legal.svelte](https://github.com/iotaledger/firefly/blob/develop/packages/shared/routes/setup/Legal.svelte)
- [x] Update privacy policy on the Firefly website

### Acceptance criteria

All requirements completed

### Creation checklist

- [X] I have assigned this task to the correct people
- [X] I have added the most appropriate labels
- [x] I have linked the correct milestone and/or project

### PR DESCRIPTION
**Pull request by rajivshah3**
## Summary
- Update address in ToS and privacy policy
- Update punctuation in privacy policy
- Add Sentry and Cloudflare to privacy policy

### Changelog
```
Update Terms of Service and Privacy Policy
```

## Relevant Issues
Closes #2262 

## Type of Change
- [ ] __Breaking__ - any change that would cause existing functionality to not work as expected
- [x] __Chore__ - refactoring, build scripts or anything else that isn't user-facing
- [ ] __Docs__ - changes to the documentation
- [ ] __Fix__ - a change which fixes an issue
- [ ] __New__ - a change which implements a new feature
- [x] __Update__ - a change which updates existing functionality

### Platforms
Please select any platforms where your changes have been tested.
- __Desktop__
	- [x] MacOS
	- [ ] Linux
	- [ ] Windows
- __Mobile__
	- [ ] iOS
	- [ ] Android

### Instructions
View the ToS and privacy policy screens during onboarding

## Checklist
Please tick all of the following boxes that are relevant to your changes, and delete those that are not.
- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have verified that my latest changes pass CI workflows for testing and linting

### CODE REVIEW

File: packages/shared/routes/setup/Legal.svelte [hunk 1]
```diff
@@ -609,19 +609,32 @@
                     <Text type="p" secondary classes="mb-4">
                         We use CoinGeckoAPI of Gecko Labs Pte. Ltd, 101 Upper Cross Street, #05-16 People's Park Centre, Singapore
                         058357 ("CoinGecko") to obtain historical price and market data for cryptocurrencies. Further information
-                        regarding their privacy policy can be found under https://www.coingecko.com/en/privacy
+                        regarding their privacy policy can be found under https://www.coingecko.com/en/privacy.
                     </Text>
                     <Text type="h3" classes="mb-5">GitHub</Text>
                     <Text type="p" secondary classes="mb-4">
-                        We use GitHub of GitHub Inc 88 Colin P. Kelly Street, San Francisco, CA 94107, USA ("GitHub" - in the
+                        We use GitHub of GitHub Inc., 88 Colin P. Kelly Street, San Francisco, CA 94107, USA ("GitHub" - in the
```
Comment (begonaalvarezd) [inline packages/shared/routes/setup/Legal.svelte:616]: We have to remove `"GitHub" -` from here

File: packages/shared/routes/setup/Legal.svelte [hunk 2]
```diff
@@ -609,19 +609,32 @@
                     <Text type="p" secondary classes="mb-4">
                         We use CoinGeckoAPI of Gecko Labs Pte. Ltd, 101 Upper Cross Street, #05-16 People's Park Centre, Singapore
                         058357 ("CoinGecko") to obtain historical price and market data for cryptocurrencies. Further information
```
Comment (begonaalvarezd) [inline packages/shared/routes/setup/Legal.svelte:611]: We have to remove `("CoinGecko")`

File: packages/shared/routes/setup/Legal.svelte (hunk 1 above)
Comment (begonaalvarezd) [inline packages/shared/routes/setup/Legal.svelte:616]:
```suggestion
                        We use GitHub of GitHub Inc., 88 Colin P. Kelly Street, San Francisco, CA 94107, USA (in the
```

File: packages/shared/routes/setup/Legal.svelte (hunk 2 above)
Comment (begonaalvarezd) [inline packages/shared/routes/setup/Legal.svelte:611]:
```suggestion
                        058357 to obtain historical price and market data for cryptocurrencies. Further information
```

File: packages/shared/routes/setup/Legal.svelte (hunk 1 above)
Comment (rajivshah3) [reply packages/shared/routes/setup/Legal.svelte:616]: Ah good catch, I forgot about that change. Thanks!

**Comment by rajivshah3** [comment]
Awaiting confirmation from the legal team regarding whether the ToS needs to be updated

**Review by nicole-obrien — APPROVED** [review]
👍

**Pull request merged**